tools/accuracy_checker/accuracy_checker/adapters/centernet.py

```diff
@@ -0,0 +1,161 @@
+            A_w = A_w.reshape(-1, *kernel_size)
+
+            return A_w.max(axis=(1, 2)).reshape(output_shape)
+
+        pad = (kernel - 1) // 2
+
+        hmax = np.array([max_pool2d(channel, kernel, pad) for channel in heat[0]])
+        hmax = np.expand_dims(hmax, axis=0)
+        keep = (hmax == heat)
+        return heat * keep
+
+    @staticmethod
+    def _transform_preds(coords, center, scale, output_size):
+        def affine_transform(pt, t):
+            new_pt = np.array([pt[0], pt[1], 1.], dtype=np.float32).T
+            new_pt = np.dot(t, new_pt)
+            return new_pt[:2]
+
+        target_coords = np.zeros(coords.shape)
+        trans = CenterNetAffineTransformation.get_affine_transform(center, scale, 0, output_size, inv=1)
+        for p in range(coords.shape[0]):
+            target_coords[p, 0:2] = affine_transform(coords[p, 0:2], trans)
+        return target_coords
+
+    @staticmethod
+    def _transform(dets, center, scale, heigth, width):
+        for i in range(dets.shape[0]):
+            dets[i, :, :2] = CTDETAdapter._transform_preds(
+                dets[i, :, 0:2], center[i], scale[i], (width, heigth))
+            dets[i, :, 2:4] = CTDETAdapter._transform_preds(
+                dets[i, :, 2:4], center[i], scale[i], (width, heigth))
+        return dets
+
+    def process(self, raw, identifiers=None, frame_meta=None):
+        result = []
+        hm = self._extract_predictions(raw, frame_meta)[self.hm]
+        wh = self._extract_predictions(raw, frame_meta)[self.wh]
+        reg = self._extract_predictions(raw, frame_meta)[self.reg]
+        heat = 1/(1 + np.exp(-hm))
+        batch, cat, height, width = heat.shape
+
+        heat = self._nms(heat)
+        scores, inds, clses, ys, xs = self._topk(heat, K=100)
+        reg = self._tranpose_and_gather_feat(reg, inds)
+        K = 100
+
+        reg = reg.reshape((batch, K, 2))
+        xs = xs.reshape((batch, K, 1)) + reg[:, :, 0:1]
+        ys = ys.reshape((batch, K, 1)) + reg[:, :, 1:2]
+
+        wh = self._tranpose_and_gather_feat(wh, inds)
+        wh = wh.reshape((batch, K, 2))
+        clses = clses.reshape((batch, K, 1)).astype(np.float)
+        scores = scores.reshape((batch, K, 1))
+        bboxes = np.concatenate((xs - wh[..., 0:1] / 2,
+                                 ys - wh[..., 1:2] / 2,
+                                 xs + wh[..., 0:1] / 2,
+                                 ys + wh[..., 1:2] / 2), axis=2)
+        detections = np.concatenate((bboxes, scores, clses), axis=2)
+        detections = np.array(detections)
+        im_size = frame_meta[0].get('image_size')
+        scale = max(im_size)
+        center = np.array(im_size[:2])/2.0
+        dets = self._transform(detections, [np.flip(center)], [scale], height, width)
+        x_min, y_min, x_max, y_max, scores, classes = dets[0].transpose(1, 0)
+        result.append(DetectionPrediction(identifiers, classes, scores, x_min, y_min, x_max, y_max))
```

Comment:
identifiers, it is list of identifier, if model support only batch size 1, you need extract it
```python
identifiers[0]
```
if model can be evaluated with batch >1, output processing should be done in the cycle (one DetectionPrediction for one image in the batch)